components/ProductsList.vue

```diff
@@ -34,7 +34,7 @@
 
 <script setup lang="ts">
 import { FullProduct } from '~/types/products';
-import ArrowIcon from '@/assets/icons/arrow_thin_left.svg?skipsvgo';
+// import ArrowIcon from '@/assets/icons/arrow_thin_left.svg?skipsvgo';
```

Comment:
This line can be safely removed.

As well a lines [5-12] if you are there, pretty please

Reply:
Done ✅